K8s-deployment/Charts/gis-interface/templates/ingress/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: gis-interface
+  namespace: {{ .Values.nameSpace}}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: ingress
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  
+  annotations:
+  {{- if or .Values.ingress.annotations .Values.commonAnnotations }}
+    {{- if .Values.ingress.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.ingress.annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+    nginx.ingress.kubernetes.io/limit-burst-multiplier: "1"
```

Comment:
Do not hardcode ingress annotations in template. Its given from values. The rate limit values might need to be adjusted as well other server-snippet code.